server-common/src/main/java/com/datastrato/gravitino/server/web/JettyServerConfig.java

```diff
@@ -177,6 +177,76 @@ public final class JettyServerConfig {
           .version("0.4.0")
           .stringConf()
           .createWithOptional();
+  public static final ConfigEntry<Boolean> ENABLE_CORS_FILTER =
+      new ConfigBuilder("enableCorsFilter")
+          .doc("Enable cross origin resource share filter")
+          .version("0.4.0")
+          .booleanConf()
+          .createWithDefault(false);
+
+  public static final ConfigEntry<String> ALLOWED_ORIGINS =
+      new ConfigBuilder("allowedOrigins")
+          .doc(
+              "A comma separated list of origins that are allowed to access the resources. Default value is *, meaning all origins")
+          .version("0.4.0")
+          .stringConf()
+          .createWithDefault("*");
+
+  public static final ConfigEntry<String> ALLOWED_TIMING_ORIGINS =
+      new ConfigBuilder("allowedTimingOrigins")
+          .doc(
+              "A comma separated list of origins that are allowed to time the resource. Default value is the empty string, meaning no origins.")
+          .version("0.4.0")
+          .stringConf()
+          .createWithDefault("");
+
+  public static final ConfigEntry<String> ALLOWED_METHODS =
+      new ConfigBuilder("allowedMethods")
+          .doc(
+              "a comma separated list of HTTP methods that are allowed to be used when accessing the resources. Default value is GET,POST,HEAD,DELETE")
+          .version("0.4.0")
+          .stringConf()
+          .createWithDefault("GET,POST,HEAD,DELETE");
+
+  public static final ConfigEntry<String> ALLOWED_HEADERS =
+      new ConfigBuilder("allowedHeaders")
+          .doc(
+              "A comma separated list of HTTP headers that are allowed to be specified when accessing the resources. Default value is X-Requested-With,Content-Type,Accept,Origin. If the value is a single *, this means that any headers will be accepted.")
```

Comment:
The code line seems too long, is it ok for spotless check?

Reply:
Yes, I used spotless to format the file.

Reply:
I would suggest to limit the line length to 100 words.